src/utils/createGridError.ts

```
@@ -0,0 +1,6 @@
+import type { GridApi } from 'ag-grid-community'
+
+export const createGridError = (gridApi: GridApi, errorMessage: string) => {
+  gridApi.setGridOption('overlayNoRowsTemplate', errorMessage)
+  gridApi.showNoRowsOverlay()
+}
```

in real app we need to show/hide this error conditionally so probably we need another func to hide it, much overhead

I think composable taking gridApi as an argument, and exposing hide/show would fit here better